src/APIServer/bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
```

Comment:
Is this file necessary, or is it automatically generated? if it is automatically generated, should we be committing it?

Reply:
It has been generated as part of the default ExpressJS template and is the "default execution script". I think it is worth leaving in. We do need some standard execution path, port setting and others.

Reply:
Oh ok, so it is autogenerated on run? It's probably pulling from the config anyway then (see my earlier comment).

Reply:
No. In the earlier comment you refer to there is a false dichotomy: Automatically generated or useful. It can be automatically generated and useful. It has been generated by the ExpressJS project template generator when the project was created and is supposed to be the "executable" for the project. If it is deleted we are not getting it back and I cannot easily rewrite its contents from scratch.

Reply:
Ah I see. Thanks